sbt-dotty/src/dotty/tools/sbtplugin/DottyPlugin.scala

```diff
@@ -434,7 +437,7 @@ object DottyPlugin extends AutoPlugin {
             // Apply withDottyCompat to the dependency on scalajs-test-bridge
             .map { moduleID =>
               if (moduleID.organization == "org.scala-js" && moduleID.name == "scalajs-test-bridge")
-                moduleID.withDottyCompat(scalaVersion.value)
+                moduleID.cross(CrossVersion.for3Use2_13)
```

Comment:
`sbt-dotty` is currently built with sbt 1.4.9 to avoid breaking the compatibility.
What would be the point of releasing for sbt 1.5.0 only?

Reply:
Ah that's true, I changed it to avoid an error in the build due to the deprecation warning, but maybe we can silence that warning or remove -Xfatal-warnings here